ci/matrix.yaml

```
@@ -54,6 +54,7 @@ workflows:
     - {jobs: ['test'],  project: 'cudax', ctk: ['curr'], std: 20,    cxx: ['gcc', 'clang', 'msvc'], gpu: 'rtx2080'}
     # Python and c/parallel jobs:
     - {jobs: ['test'], project: ['cccl_c_parallel'], gpu: ['rtx2080', 'l4', 'h100']}
+    - {jobs: ['test'], project: ['cccl_c_stf'], gpu: ['rtx2080', 'l4', 'h100']}
```

nitpick: Try to keep things aligned for better readability.

```suggestion
    - {jobs: ['test'], project: ['cccl_c_stf'],      gpu: ['rtx2080', 'l4', 'h100']}
```